MediaManager/Plugin.MediaManager.Abstractions/Implementations/MediaManagerBase.cs

```diff
@@ -255,7 +291,8 @@ private void OnPlayingChanged(object sender, PlayingChangedEventArgs e)
         {
             if (sender != CurrentPlaybackManager) return;
             MediaFinished?.Invoke(sender, e);
-            await PlayNext();
+            if (MediaQueue.Repeat == RepeatType.RepeatOne) await Seek(TimeSpan.Zero);
```

Comment:
Yes, but this should be handled in the PlayNext() method. Otherwise you again have duplicate code or you do the wrong thing when the user then hit's the "next" button in your UI.